### ISSUE TITLE
Unable to start the Grafana docker container

### ISSUE THREAD

**Issue by mbwhite** [hyperledger/caliper-benchmarks#26]
Following error is present in Grafana logs

```
t=2019-10-21T09:31:54+0000 lvl=eror msg="Can't read alert notification provisioning files from directory" logger=provisioning.notifiers path=/etc/grafana/provisioning/notifiers error="open /etc/grafana/provisioning/notifiers: no such file or directory"
t=2019-10-21T09:31:54+0000 lvl=eror msg="Server shutdown" logger=server reason="Service init failed: Failed to create provisioner: Failed to read dashboards config: could not parse provisioning config file: docker-and-system.yaml error: yaml: unmarshal errors:\n  line 1: cannot unmarshal !!map into []*dashboards.DashboardsAsConfigV0"
```

**Comment by mbwhite** [comment]
The `docker-and-system.yaml` file is JSON,, so needs the correct extentions

**Comment by nklincoln** [comment]
closed by #27
